application/app/src/main/java/uk/nhs/digital/uec/api/controller/LoginController.java

```diff
@@ -0,0 +1,32 @@
+package uk.nhs.digital.uec.api.controller;
```

Comment:
Lets move this into 'authentication' section

Reply:
done